packages/o/openssl/xmake.lua

```diff
@@ -32,15 +32,18 @@ package("openssl")
             if package:is_plat("android") and is_subhost("windows") and os.arch() == "x64" then
                 -- when building for android on windows, use msys2 perl instead of strawberry-perl to avoid configure issue
                 package:add("deps", "msys2", {configs = {msystem = "MINGW64", base_devel = true}, private = true})
-            elseif is_subhost("windows") and not package:is_precompiled() then
```

Comment:
and why not remove `not package:is_precompiled()`? 
If you are using the build-artifacts binary, you do not need to compile openssl and install its build dependencies.

Reply:
and here

Reply:
OK, there are no `is_precompiled` now

Reply:
> OK, there is no `is_precompiled` now

but we need is_precompiled.

Reply:
Sorry, I misunderstood.

Reply:
If you are downloading a precompiled binary, then all build dependencies do not need to be installed.

https://github.com/xmake-mirror/build-artifacts/releases?q=openssl&expanded=true

Reply:
I guess the outer `package:is_precompiled()` might work as well.

https://github.com/xmake-io/xmake-repo/blob/a7144ca9afa10b5fc0c9b584ea2f96231ea53dde/packages/o/openssl/xmake.lua#L30-L31